pkg/collectors/service.go

```diff
@@ -53,12 +53,40 @@ var (
 		nil,
 	)
 
+	descServiceExternalName = metrics.NewMetricFamilyDef(
+		"kube_service_external_name",
+		"Service external name",
+		append(descServiceLabelsDefaultLabels, "external_name"),
+		nil,
+	)
+
+	descServiceLoadBalancerIP = metrics.NewMetricFamilyDef(
```

Comment:
`descServiceLoadBalancerIP` is defined here but not used anywhere. Am I missing something?